### ISSUE TITLE
Scene and oscillator keybinds should be announced

### ISSUE THREAD

**Issue by Lo-lo78** [surge-synthesizer/surge#6591]
**Is your feature request related to a problem? Please describe!**

I am frustrated when I am not sure which scene I am in or which oscillator I have selected if I am in another section of the synth.
While using Surge I would very much like the alt + s shortcuts for scenes. Alt + 1/2/3 for oscillators. used speech synthesis to tell what the status of the selection is.
Alt + s scene 1 / scene2. Alt + 1/2/3 changes the oscillator. If you are in another section like LFO I don't know after minutes if I am in one scene or another. Also for the oscillator.

**Describe the solution you'd like:**

It would be very useful while working if the screen reader would announce the selected scene or oscillator via the shortcuts.

**Describe alternatives you've considered:**

If it is easier to develop. For shortcuts, the Windows sapi 5 system speech is also good. More useful if the screen reader announced it.
Thank you!

**Comment by baconpaul** [comment]
the solution to this we decided on discord was some "Alt-I" or some such which speaks the entire state of the editor.

**Comment by Lo-lo78** [comment]
@baconpaul 
Yes exactly!
👍

**Comment by baconpaul** [comment]
Turning back to these in the new year.

In the next nightly in an couple of hours, `Alt-zero` will speak out a sentence with your patch name, selected scene, selected oscillator, selected modulator, and selected fx.

Happy to tweak the spoken text once you all try it.

**Comment by mkruselj** [comment]
Does it cost us much to add announcements when actuating those individual keybinds that change focus of various tabs?

**Comment by baconpaul** [comment]
when you actually do the switch it already announces! 

this is a key bind in case you forget where you are so you don't have to tab around the entire UI
